### ISSUE TITLE
ezSerilacli Not working in Linux and Windows Works fine in MAC

### ISSUE THREAD

**Issue by aterry35** [ezflash/ezFlashCLI#29]
The ezSerialCli  while trying to program a DA14531 USB dev kit , works fine load is sucess in MAC with M1 chip,  but same script fails in Linux and Windows, it fails in the bellow step

**DEBUG:da1469xSerialLoader:Failed to get data ACK 0 101**

Can someone tell me what can cause the problem in linux and windows

**Comment by ezflash** [comment]
Did you try with the option:

```console
 -o, --one      setup 1-wire mode
````

**Comment by aterry35** [comment]
Yes I am trying with one wire mode only , With option -o Looks like in windows and Linux a timeout happens in between the read and write which happens with the len(data).I tried increasing the timeout still no improvement. Thanks and Regards Terry On Feb 3, 2023, at 12:00 AM, ezflash ***@***.***> wrote:﻿
Did you try with the option:
 -o, --one      setup 1-wire mode

**Comment by ezflash** [comment]
Expected to be fixed on [this release](https://github.com/ezflash/ezFlashCLI/releases/tag/v1.0.11)
